learn/_components/LearningList.tsx

```diff
@@ -0,0 +1,35 @@
+import { TutorialIcon } from "./TutorialIcon.tsx";
+import { ExampleIcon } from "./ExampleIcon.tsx";
+import { VideoIcon } from "./VideoIcon.tsx";
+
+export function LearningList(
+  props: {
+    title: string;
+    items: ({ label: string; id: string; type: string })[];
+  },
+) {
+  return (
+    <section className="break-inside-avoid-column">
+      <h2 className="text-lg font-semibold mb-3">{props.title}</h2>
+      <ul className="mb-8">
+        {props.items.map((item) => (
+          <li
+            className="leading-7 learning-list-item"
+            data-category={item.type}
+          >
+            <a
+              className="homepage-link mb-1 runtime-link"
+              href={item.id}
+            >
+              {item.type === "tutorial" && <TutorialIcon color="#9d9d9d" />}
+              {item.type === "example" && <ExampleIcon color="#9d9d9d" />}
+              {item.type === "video" && <VideoIcon color="#9d9d9d" />}
+
+              {item.label}
+            </a>
+          </li>
```

Comment:
Two design notes on these list items:

1. In light mode, the link color doesn't provide enough contrast to pass a11y checks;
2. When viewing the whole big list, I feel like the icons kind of blend together. Would it make sense to differentiate the three categories by icon color, perhaps? Open to other ideas, and this doesn't need to block; just noticed that since every item has an icon, they visually struck me just as bullets and I ignored them until I looked closer.